include/sdf/Element.hh

```diff
@@ -848,10 +1064,23 @@ namespace sdf
   ///////////////////////////////////////////////
   template<typename T>
   bool Element::Set(const T &_value)
+  {
+    sdf::Errors errors;
+    bool result = this->Set<T>(_value, errors);
+    for(auto& error : errors)
+    {
+        sdferr << error.Message();
+    }
```

Comment:
Use `throwOrPrintErrors`

Reply:
Added in https://github.com/gazebosim/sdformat/pull/1141/commits/8d9f1516267e36903b73e9aed7bfee371b45b1a6.